dead.py

```diff
@@ -17,21 +17,26 @@
 from identify.identify import tags_from_path
 
 UsageMap = DefaultDict[str, Set[str]]
+DefineSet = Set[Tuple[str, str, int]]
+DisableSet = Set[Tuple[str, int]]
+
 # https://github.com/python/typed_ast/blob/55420396/ast27/Parser/tokenizer.c#L102-L104
 TYPE_COMMENT_RE = re.compile(r'^#\s*type:\s*')
 # https://github.com/python/typed_ast/blob/55420396/ast27/Parser/tokenizer.c#L1400
 TYPE_IGNORE_RE = re.compile(TYPE_COMMENT_RE.pattern + r'ignore\s*(#|$)')
 # https://github.com/python/typed_ast/blob/55420396/ast27/Grammar/Grammar#L147
 TYPE_FUNC_RE = re.compile(r'^(\(.*?\))\s*->\s*(.*)$')
+DISABLE_COMMENT_RE = re.compile(r'dead\s*:\s*disable')
 
 
 class Visitor(ast.NodeVisitor):
     def __init__(self) -> None:
         self.filename = ''
         self.is_test = False
         self.reads: UsageMap = collections.defaultdict(set)
-        self.defines: UsageMap = collections.defaultdict(set)
+        self.defines: DefineSet = set()
```

Comment:
why did this change?

Reply:
because values from that UsageMap not used